packages/xo-server-usage-report/src/index.js

```diff
@@ -759,6 +759,12 @@ class UsageReportPlugin {
   }
 
   async _sendReport(storeData) {
+    if (this._xo.sendEmail === undefined) {
+      throw new Error(
+        'The plugin usage-report requires the plugin transport-email to be enabled'
+      )
+    }
+
```

Comment:
Make sure to add a version number to the Changelog if you need this to be released.